ua-snap/northern-climate-reports
# Epa justice demographics

Adds demographics report section!  🎺 

For local setup for testing, check out the [data-api repo at this commit](https://github.com/ua-snap/data-api/commit/f36524afc5f19002839831929f80c7fa1bfbd04d) (or newer).

Things to test:
 - Go to the Data page and review the Demographics data information
 - Choose Fairbanks and examine the data responses.
 - Choose Nabesna -- a place with 2 people.  What happens?
 - Choose Akhiok -- what happens?

If no demographic data are present, there should be a notice to the user to that effect with a "get in touch if you have questions."
